Cargo.toml

```diff
@@ -16,6 +16,46 @@ members = [
     "test-templates",
 ]
 
```

Comment:
Let's also use the [`workspace.package`](https://doc.rust-lang.org/cargo/reference/workspaces.html#the-package-table) feature to inherit common things like `contributors`, license files, version numbers, etc.

Reply:
I think everything except the package name and description can be lifted up

Reply:
Got it. `rust-version` was slightly different between crates (between 1.56 and 1.59). I set it in the workspace to 1.59, but this can of course be changed.